### DISCUSSION TITLE
Sitemap.xml

### DISCUSSION THREAD

**Discussion by AdamczykMaciej** [community/community#106607] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question, improvement, issue

### Body

Seotoolkit Sitemap
When xhtml tags are created in my sitemap.xml then it appears as plain text in browser. When inspecting the source code it is a valid xml but in browser it looks like a plain text. I found similar threads: 
https://stackoverflow.com/questions/60564479/sitemap-xml-with-xhtmllink-rel-alternate-hreflang-href-for-multilang
https://stackoverflow.com/questions/51923627/xml-sitemap-rendering-as-plain-text

![image](https://github.com/community/community/assets/34938878/428eaea7-986f-47df-91ee-cc2cc591821d)
![image](https://github.com/community/community/assets/34938878/55c4d8a7-0606-4cfb-aba3-977338257dac)

I think it might be the issue with
https://stackoverflow.com/questions/24864254/google-multilingual-sitemap-issue

**Comment by queenofcorgis** [comment]
Hi @AdamczykMaciej ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google or the project you found the source code in . It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by AdamczykMaciej** [comment]
Ok, I will move it. Thank you :)
